### ISSUE TITLE
Phosh quick menu System Volume slider disabled

### ISSUE THREAD

**Issue by Torbuntu** [nikhiljha/pp-fedora-sdsetup#14]
In the drop down quick menu on Phosh the system volume slider is missing the knob to adjust volume. 

This makes it impossible to change volume from Phosh.

**Comment by thibaultamartin** [comment]
@Torbuntu IIRC you have a partial answer to that

**Comment by Torbuntu** [comment]
```
Jun 26 16:02:45 localhost.localdomain phosh[1248]: gvc_mixer_control_get_vol_max_norm: assertion 'GVC_IS_MIXER_CONTROL (control)' failed
```

This is all I really know. Looks like something to do with gvc, but that is the bulk of where I got with this issue.

**Comment by Torbuntu** [comment]
I see that the libgnome-volume-control commit on the phosh repo is a few releases behind Fedora. 

A few commits ahead of the one they use is this: https://gitlab.gnome.org/GNOME/libgnome-volume-control/-/commit/3f457453f031a2936ff267348107e6513be3aa27 which looks like it added the error I am seeing. This doesn't mean that it created the error, but maybe just exposed it. Still unsure what is missing/broken here.

I updated the libgnome-volume-control to the latest commit on the Phosh repo, which did not solve anything.

**Comment by Torbuntu** [comment]
more errors when playing with sound in gnome settings:
```
Jun 26 22:35:03 localhost.localdomain phosh[1034]: instance with invalid (NULL) class pointer
Jun 26 22:35:03 localhost.localdomain phosh[1034]: g_signal_handlers_disconnect_matched: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed
```

**Comment by Torbuntu** [comment]
I see in phosh's `settings/gvc-channel-bar.c`  https://source.puri.sm/Librem5/phosh/-/blob/master/src/settings/gvc-channel-bar.c this define:
```
#define ADJUSTMENT_MAX_NORMAL gvc_mixer_control_get_vol_max_norm(NULL)
```

Which corelates, at least in name, to this error I see in `journalctl -e | grep phosh` 
```
Jun 26 16:02:45 localhost.localdomain phosh[1248]: gvc_mixer_control_get_vol_max_norm: assertion 'GVC_IS_MIXER_CONTROL (control)' failed
```

**Comment by Torbuntu** [comment]
Solved by reverting the libgnome-volume-control dependency to the same version used in the phosh repository. Updated the spec file to fix it.

**Comment by Torbuntu** [comment]
Fixed in phosh-0.3.1-3
